wallet/wallet.go

```diff
@@ -471,6 +471,12 @@ func (w *Wallet) scanChain(startHeight int32,
 			Checkpoints[len(w.chainParams.Checkpoints)-1].Height
 	}
 	isCurrent := func(bestHeight int32) bool {
+		// If the best height is zero, we assume the chain backend
+		// still is looking for peers to sync to.
```

Comment:
Not just peers, but in the full node case, this would ensure we wait for a chain to exist when running on regtest or simnet.